topostats/plotting.py

```diff
@@ -272,9 +278,63 @@ def _set_label(self, var: str):
         self.label = self.var_to_label[var]
         LOGGER.debug(f"[plotting] self.label     : {self.label}")
 
-    def summarise_by_image(self):
-        """Summarise statistics by image."""
-        self.summary_data = self.df.groupby(["image", "threshold"]).describe()
+    @staticmethod
+    def get_relative_paths(paths: List[Path]) -> list:
```

Comment:
Trying to think ahead about typing the return value could also be...

```python
List[str]
```

At some point I'd like to introduce [mypy](https://mypy.readthedocs.io) into the pre-commit hooks and then retrospectively apply it across the code base and then we catch these as we go.

Reply:
Done 👍